### DISCUSSION TITLE
[Feature] raven does not support self-signed certificates

### DISCUSSION THREAD

**Discussion by beltiras** [saltstack/salt#67517] [Feature Requests]
**Description**
For anyone using self-signed certificates for their on-site Sentry deployment, you can not use the built-in configuration of dropping in a SENTRY_DSN and an alternate ca-certificates.crt, getting error logging quickly with ease.

All of this is of course due to Python not using system ca-certificates.crt (like it should). I searched the issues and tried to google it a bit but could not find anything except the trick of appending ?verify_ssl=0 to the DSN. This is not an acceptable long-term solution.

I've not found any issues, open or closed (in this repo), that tackles the issue. Searching the repository apparently raven is not part of dependencies. Is there a reason why upgrading to sentry_sdk is hard?

**Comment by trickvi** [comment]
Raven does support it with the `?ca_certs=path/to/certs` 
Still that doesn't change the fact that raven is deprecated (sentry_sdk should be used instead)

**Comment by alexey-zhukovin** [comment]
@arnists thank you for your question.
I think this is not in priority task.
I can create feature request.

**Comment by beltiras** [comment]
Any attention to this issue is appreciated. Where would I track the feature request?

**Comment by alexey-zhukovin** [comment]
Feature request:
We have  `sentry_return` 
salt/returners/sentry_return.py
It uses `raven` library. This library is outdated.
`https://docs.sentry.io/clients/python/api/`
Update or create another return with `sentry_sdk` support.
https://docs.sentry.io/error-reporting/quickstart/?platform=python
